recipes/pciutils/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class PciUtilsConan(ConanFile):
+    name = "pciutils"
+    license = "BSD-3-Clause"
+    description = "The PCI Utilities package contains a library for portable access to PCI bus"
+    topics = ("pci", "pci-bus", "hardware", "local-bus")
+    homepage = "https://github.com/pciutils/pciutils"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_zlib": [True, False], "with_udev": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_zlib": True, "with_udev": False}
+
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Platform {} is currently not supported by this recipe".format(self.settings.os))
```

Comment:
The documentation says that Windows is possible, but needs `WinIO.dll` dependency.
```suggestion
            # FIXME: missing WinIO.dll for Windows support
            raise ConanInvalidConfiguration("Os {} is currently not supported by this recipe".format(self.settings.os))
```

Reply:
The windows port builds but it is no longer supported I have not tested to see if it works

Reply:
darwin is also listed as supported

Reply:
If it builds, then I keep it.
It's good CI for the upstream project and if somebody runs into issues, they can always ask upstream.
Often upstream don't support Windows because they don't test it on CI, don't use it, or they assume nobody on Windows is smart enough to use it :smile:

Reply:
Next PR, my main goal is to added some necessities for my team. I'll add it to my list of partial support there'll be another lock down here soon so I'll have time!

Reply:
Perfect for the todo, not so perfect for the lockdown.
I remember packaging this one in the past for openmpi. But I lost it somewhere along the way apparently.